src/applications/facility-locator/config.js

```diff
@@ -25,23 +25,19 @@ export const api = {
  * enable Community Care Provider Locator features of the
  * existing Facility Locator App.
  */
-export const ccLocatorEnabled = () => {
-  return true;
-};
+export const ccLocatorEnabled = () => true;
 
-/* eslint-disable camelcase */
 export const facilityTypes = {
-  all: 'All Facilities',
-  va_health_facility: 'VA Health',
-  va_cemetery: 'Cemetery',
-  va_benefits_facility: 'Benefits',
-  vet_center: 'Vet Center',
-  health: 'VA Health',
-  cc_provider: 'Community Care (Non-VA Health)',
-  cemetery: 'Cemetery',
-  benefits: 'Benefits',
+  [LocationType.ALL]: 'All Facilities',
+  [FacilityType.VA_HEALTH_FACILITY]: 'VA Health',
+  [FacilityType.VA_CEMETARY]: 'Cemetery',
+  [FacilityType.VA_BENEFITS_FACILITY]: 'Benefits',
+  [FacilityType.VET_CENTER]: 'Vet Center',
+  [LocationType.HEALTH]: 'VA Health',
+  [LocationType.CC_PROVIDER]: 'Community Care (Non-VA Health)',
+  [LocationType.CEMETARY]: 'Cemetery',
+  [LocationType.BENEFITS]: 'Benefits',
```

Comment:
I got rid of the `eslint-disable camelcase` by leveraging other constants, which also added some meaning to the data structure.